File: smart-meal-back/smart-meal-back/Contracts/Restaurant.cs
namespace smart_meal_back.Contracts;

public class Restaurant
{
    public int? RestaurantId { get; set; }
    public string Name { get; set; }
    public string? Illustration { get; set; }
    public string Address { get; set; }
    public string? Siret { get; set; }
    public string? Tva { get; set; }
    public AdminAccount Admin { get; set; }
    public string? PaymentId { get; set; }
    public string?  ContractUrl { get; set; }
}

File: smart-meal-front/src/domain/model/Restaurant.ts
import {Administrator} from "./Administrator";

export interface Restaurant {
  isVisible?: boolean;
  restaurantId?: string,
  name: string,
  illustration: string
  address: string
  siret: string
  tva: string
  categoriesOfItemAvailable?: string[]
  admin?: Administrator
  averagePrice?: number
  contractUrl?: string
  paymentId?: string
}
